### ISSUE TITLE
The robot may start oscillating visibly right after starting the walking

### ISSUE THREAD

**Issue by S-Dafarra** [robotology/walking-controllers#84]
We experienced this during a demo with the robot that started to oscillate visibly from left to right. It would nice to investigate possible causes and solutions.

cc @isorrentino

**Comment by isorrentino** [comment]
The last week I launched the walking several time and I acquired data for each one. I used https://github.com/robotology/walking-controllers#how-to-dump-data to log data. I cannot upload the datasets as the file is too big.
Here the datasets:
[walkingStartupData.zip](https://github.com/robotology/walking-controllers/files/6180542/walkingStartupData.zip)

I still need to analyze them, but in general, I can say that the robot did not repeat the behaviour we had in Milan. Just once it started oscillating but much much less and actually I'm not sure I can see something from the data.
